modules/proxistoreBidAdapter.js

```diff
@@ -1,21 +1,22 @@
 const { registerBidder } = require('../src/adapters/bidderFactory');
 const BIDDER_CODE = 'proxistore';
+const PROXISTORE_VENDOR_ID = 418;
 
-function _getFormatSize(sizeArr) {
-  return {
-    width: sizeArr[0],
-    height: sizeArr[1]
-  }
+function _mapSizes(sizes) {
+  return sizes
+    .flatMap(array2d => array2d
```

Comment:
Hi @vincentproxistore 
When I was retesting the adapter, I ran our browserstack tests locally and found an issue with this `flatMap` function.  It is not supported by IE or Edge.  

If you want to continue to use this function, you'll need to add a polyfill.  There appears to be one in `core-js`; should be able to import it from `core-js/library/fn/array/flat-map`.   Otherwise you need to implement another form of logic that's browser compatible.